.gitignore

```diff
@@ -1 +1,3 @@
 target/
+
+.idea/
```

Comment:
IDE-specific ignores should not be put in a project's `.gitignore` file. They should be in [your global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).